internal/connect/eula.go

```diff
@@ -199,22 +199,19 @@ func AcceptEULA() error {
 		return nil
 	}
 
-	// If we encounter a switch of base products (e.g. SLES -> SLES_SAP)
-	// we can not fetch the product information and thus the EULA since there
+	// Skip if we can not fetch the product information and thus the EULA since there
 	// might be no registration in place right now.
 	// See bsc#1218649 and bsc#1217961
-	if base.ToTriplet() != CFG.Product.ToTriplet() {
-		return nil
-	}
-
 	prod, err := showProduct(base)
 	if err != nil {
-		return err
+		Debug.Print("Cannot load base product details for EULA.")
+		return nil
```

Comment:
Why not return `err` here?

Reply:
If we return `err` here, the process will fail when the base product cannot get loaded, for example because the system is not yet registered, or it is switching the base product